### DISCUSSION TITLE
process_batch 函数疑问

### DISCUSSION THREAD

**Discussion by duzijj** [zai-org/ChatGLM3#967] [General]
[finetune_demo](https://github.com/THUDM/ChatGLM3/tree/main/finetune_demo)
/finetune_hf.py   
这里的process_batch函数中
input_ids.append(tokenizer.eos_token_id)
loss_masks = [False, *loss_masks]
为什么loss_masks添加的False要放前面而不是放在末尾？

**Comment by zRzRzRzRzRzRzR** [comment]
@zhangch9
